src/modules/render.js

```diff
@@ -0,0 +1,19 @@
+const list = document.querySelector('tbody');
+
+const render = () => {
+  fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/HW6Rm5sf6ZsLRuJNGtLy/scores/').then((response) => response.json()).then((data) => {
+    list.innerHTML = '';
+    for (let i = 0; i < data.result.length; i += 1) {
+      const listItem = document.createElement('tr');
+      listItem.innerHTML = `
+            <td>${data.result[i].user}</td>
+            <td>${data.result[i].score}</td>
+        `;
+      list.appendChild(listItem);
+    }
+  }).catch((err) => {
+    console.log('rejected', err);
+  });
+};
```

Comment:
- [ ] Your project is working fine good job :clap: but in this function, you should use `async` and `await` JavaScript features to consume the API. :slightly_smiling_face: 
- [ ] Use of `async` and `await` enables the use of ordinary `try/catch` blocks around asynchronous code. Note: The `await` keyword is only valid inside `async` functions within regular JavaScript code. If you use it outside of an async function's body, you will get a `SyntaxError`. :+1: 
- [ ] Below check my suggestions :point_down: 

```suggestion
const render = async () => {
  try {
    const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/HW6Rm5sf6ZsLRuJNGtLy/scores/');
    const data = await response.json();
    
    list.innerHTML = '';
    for (let i = 0; i < data.result.length; i += 1) {
      const listItem = document.createElement('tr');
      listItem.innerHTML = `
            <td>${data.result[i].user}</td>
            <td>${data.result[i].score}</td>
        `;
      list.appendChild(listItem);
    }
  } catch (error) {
    console.log('rejected', error);
  }
};
```
- [ ] The `render` function is now an `async` function, which allows us to use `await` when making asynchronous requests. We use `await` when fetching data from the API and parsing the response as JSON data.

- [ ] We also replaced the `.then()` and `.catch()` methods with a `try...catch` block to handle errors more elegantly.